sycl/test/basic_tests/args-size-overflow.cpp

```diff
@@ -1,21 +0,0 @@
-// RUN: %clangxx -fsycl -fsycl-device-only -Xclang -verify -Wsycl-strict %s -fsyntax-only
```

Comment:
Why was this test deleted?

Reply:
This test was added to make sure that the warning is emitted for SYCL headers which are included as system headers. It turned out that it is possible to emulate such include scheme in FE tests as well, so we decided to remove this test to avoid duplication, because we are going to change FE tests to use "system headers include scheme".